ios/RNMessageStream.m

```diff
@@ -126,6 +161,10 @@ - (BOOL)shouldPresentInAppNotificationForMessage:(MARMessage *)message {
     }];
 }
 
+RCT_EXPORT_METHOD(acknowledgeEvent) {
+    self.showDefaultInAppNotification = NO;
```

Comment:
The `showDefaultInAppNotification` value is never reset, so the wait will finish immediately after the first in-app notification is handled. I would prefer a situation where the RN side can send down a boolean indicating whether or not the in-app notification was handled, which is then returned to the native SDK.